v4/newrelic/segments.go

```diff
@@ -112,6 +115,10 @@ type ExternalSegment struct {
 	// external metrics and the "component" span attribute.  It should be
 	// the framework making the external call.
 	Library string
+
+	// statusCode is the status code for the response.  This value takes
+	// precedence over the status code set on the Response.
```

Comment:
This comment is fairly confusing on first read-through, although after I slow down and re-read it a couple of times I understand.  Maybe something like this would be a little more readable?

> statusCode represents the status code that will be reported as the response to this external call.  This value takes precedence over the status code set on the `Response` field.